inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp

```diff
@@ -7448,3 +7449,196 @@ INSTANTIATE_TEST_CASE_P(DISABLED_fusings_gpu,
                             reduce_test_params{CASE_REDUCE_I8_3, 2, 4, reduce_mode::mean, {reduce::along_x}, true, "reduce_ref"},
                             reduce_test_params{CASE_REDUCE_U8_3, 2, 4, reduce_mode::l2, {reduce::along_x}, true, "reduce_ref"}
                         }), );
+
+
+/* ----------------------------------------------------------------------------------------------------- */
+/* ------------------------------------------ ScatterNDUpdate cases ------------------------------ */
+/* ----------------------------------------------------------------------------------------------------- */
+
+struct scatter_nd_update_test_params {
+    tensor input_shape;
+    tensor indices_shape;
+    tensor updates_shape;
+    int max_number_in_indices;
+    int indices_rank;
+    data_types data_type;
+    format input_format;
+    data_types default_type;
+    format default_format;
+    size_t expected_fused_primitives;
+    size_t expected_not_fused_primitives;
+};
+
+
```

Comment:
there are cases where indices' ranks are <= 2d 
could we add >=3d indices?

Reply:
I fixed it.

Reply:
where are they?

Reply:
![image](https://user-images.githubusercontent.com/70923720/109599437-872dcc80-7b5e-11eb-9321-55f19c857e96.png)